app/src/main/java/com/kickstarter/ApplicationModule.java

```diff
@@ -639,7 +639,7 @@ ExperimentsClientType provideOptimizely(final @ApplicationContext @NonNull Conte
     final OptimizelyManager optimizelyManager = OptimizelyManager.builder()
       .withSDKKey(optimizelyKey)
       .withDatafileDownloadInterval(60L * 10L)
-      .withEventDispatchInterval(60L)
+      .withEventDispatchInterval(2L)
```

Comment:
Cool! Is the goal of the event dispatch interval to dispatch events by batch?

Reply:
Yeah, that's the way `Optimizely X` for Android handles dispatching events https://docs.developers.optimizely.com/full-stack/docs/configure-event-dispatcher-android